diff --git a/admyral/actions/integrations/communication/slack.py b/admyral/actions/integrations/communication/slack.py
index 0b96d52..3f4cf07 100644
--- a/admyral/actions/integrations/communication/slack.py
+++ b/admyral/actions/integrations/communication/slack.py
@@ -72,7 +72,7 @@ def send_slack_message(
         response_body = response.json()
         if not response_body.get("ok"):
             raise RuntimeError(
-                f"Failed to send message in Slack. Error: {response_body.get("error")}"
+                f"Failed to send message in Slack. Error: {response_body.get('error')}"
             )
         return response_body
 
diff --git a/admyral/models/condition.py b/admyral/models/condition.py
index 8cb4092..7ec1798 100644
--- a/admyral/models/condition.py
+++ b/admyral/models/condition.py
@@ -99,7 +99,7 @@ class OrConditionExpression(ConditionExpression):
         return visitor.visit(self)
 
 
-type Condition = Annotated[
+Condition = Annotated[
     ConstantConditionExpression
     | UnaryConditionExpression
     | BinaryConditionExpression
diff --git a/admyral/models/editor.py b/admyral/models/editor.py
index 5bca044..1c6ec39 100644
--- a/admyral/models/editor.py
+++ b/admyral/models/editor.py
@@ -73,7 +73,7 @@ class EditorWorkflowIfNode(EditorWorkflowBaseNode):
     condition: str
 
 
-type EditorWorkflowNode = (
+EditorWorkflowNode = (
     EditorWorkflowStartNode | EditorWorkflowActionNode | EditorWorkflowIfNode
 )
 
