diff --git a/langchain_postgres/vectorstores.py b/langchain_postgres/vectorstores.py
index e1630a1..cc93b2c 100644
--- a/langchain_postgres/vectorstores.py
+++ b/langchain_postgres/vectorstores.py
@@ -29,13 +29,13 @@ from langchain_core.documents import Document
 from langchain_core.embeddings import Embeddings
 from langchain_core.utils import get_from_dict_or_env
 from langchain_core.vectorstores import VectorStore
-from sqlalchemy import SQLColumnExpression, cast, create_engine, delete, func, select
-from sqlalchemy.dialects.postgresql import JSON, JSONB, JSONPATH, UUID, insert
+from sqlalchemy import Column, cast, create_engine, delete, func, select
+from sqlalchemy.dialects.postgresql import JSON, JSONB, UUID, insert
 from sqlalchemy.engine import Connection, Engine
 from sqlalchemy.ext.asyncio import (
     AsyncEngine,
     AsyncSession,
-    async_sessionmaker,
+    async_session,
     create_async_engine,
 )
 from sqlalchemy.orm import (