diff --git a/pyproject.toml b/pyproject.toml
index 6fa835a..550221b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,4 +1,3 @@
-[project]
 name = "fastmcp"
 dynamic = ["version"]
 description = "A more ergonomic interface for MCP servers"
diff --git a/src/fastmcp/__init__.py b/src/fastmcp/__init__.py
index fdbfb9d..a605858 100644
--- a/src/fastmcp/__init__.py
+++ b/src/fastmcp/__init__.py
@@ -4,5 +4,5 @@ from importlib.metadata import version
 from .server import FastMCP, Context
 from .utilities.types import Image
 
-__version__ = version("fastmcp")
+__version__ = "0.1.0"
 __all__ = ["FastMCP", "Context", "Image"]
