diff --git a/job_scraper/workday/scraper.py b/job_scraper/workday/scraper.py
index ed4c6c0..c08d19b 100644
--- a/job_scraper/workday/scraper.py
+++ b/job_scraper/workday/scraper.py
@@ -74,7 +74,7 @@ class WorkdayScraper:
             self.done_event.set()
 
     def scrape(self):
-        self.update_func(f"Scraping Workday companies:\t{", ".join(self.company_urls.keys())}")
+        self.update_func(f"Scraping Workday companies:\t{', '.join(self.company_urls.keys())}")
 
         for company_name, company_url in self.company_urls.items():
             self.driver.get(company_url)
