diff --git a/Dockerfile b/Dockerfile
deleted file mode 100644
index d991715..0000000
--- a/Dockerfile
+++ /dev/null
@@ -1,14 +0,0 @@
-# Dockerfile to build installer for Cover Agent
-FROM python:3.12-bullseye
-
-WORKDIR /app
-
-# Copy all files
-COPY . .
-
-# Install required packages
-RUN pip install poetry wandb
-RUN poetry install
-
-# Run the make installer as a CMD
-CMD ["make", "installer"]
\ No newline at end of file
diff --git a/pyproject.toml b/pyproject.toml
index 40c1a68..e84cc3b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -55,7 +55,7 @@ cover-agent-full-repo = "cover_agent.main_full_repo:main"
 generate-report = "cover_agent.UnitTestDB:dump_to_report_cli"
 
 [tool.poetry-dynamic-versioning]
-enable = true
+enable = false
 
 [tool.poetry-dynamic-versioning.from-file]
 source = "cover_agent/version.txt"
