diff --git a/src/dataformer/llms/asyncllm.py b/src/dataformer/llms/asyncllm.py
index f2b9bbc..a670346 100644
--- a/src/dataformer/llms/asyncllm.py
+++ b/src/dataformer/llms/asyncllm.py
@@ -406,6 +406,8 @@ class AsyncLLM:
                         raise ValueError("Wrong model provided for the api_provider or url. The model doesn't exist on the given api_provider or url.")
                     print("Model verified successfully")
             else:
+                if os.getenv('MOCK_API_CALLS'):
+                    return
                 raise ValueError("tried to verify the model but received error." ,response)
             
     def get_request_url(self):