diff --git a/camp_zipnerf/internal/datasets.py b/camp_zipnerf/internal/datasets.py
index c7014f8..1faee41 100644
--- a/camp_zipnerf/internal/datasets.py
+++ b/camp_zipnerf/internal/datasets.py
@@ -104,7 +104,21 @@ def convert_colmap_cam(cam):
   return pixtocam, params, camtype
 
 
-class NeRFSceneManager(pycolmap.SceneManager):
+class SceneManager:
+    def load_cameras(self):
+        pass
+
+    def load_images(self):
+        pass
+
+    def load_points3D(self):
+        pass
+
+    def __init__(self):
+        self.cameras = {}
+        self.images = {}
+
+class NeRFSceneManager(SceneManager):
   """COLMAP pose loader.
 
   Minor NeRF-specific extension to the third_party Python COLMAP loader.
