diff --git a/examples/slackbot/tools.py b/examples/slackbot/tools.py
index f32c59a..7156757 100644
--- a/examples/slackbot/tools.py
+++ b/examples/slackbot/tools.py
@@ -30,7 +30,7 @@ async def post_slack_message(
         response = await client.post(
             "https://slack.com/api/chat.postMessage",
             headers={
-                "Authorization": f"Bearer {auth_token or os.environ["SLACK_API_TOKEN"]}"
+                "Authorization": f"Bearer {auth_token or os.environ['SLACK_API_TOKEN']}"
             },
             json={
                 "channel": channel_id,