diff --git a/tnlearn/mlpclassifier.py b/tnlearn/mlpclassifier.py
index 21f3729..70123d4 100644
--- a/tnlearn/mlpclassifier.py
+++ b/tnlearn/mlpclassifier.py
@@ -382,6 +382,8 @@ class MLPClassifier(BaseModel):
         # Set the network back to training mode
         self.net.train()
 
+        # Move y_ to CPU for compatibility with predictions
+        y_ = y_.cpu().numpy()
         return accuracy_score(y_, predictions)
 
     def count_param(self):