diff --git a/lmcache/storage_backend/serde/cachegen_decoder.py b/lmcache/storage_backend/serde/cachegen_decoder.py
index 470054a..7ff8725 100644
--- a/lmcache/storage_backend/serde/cachegen_decoder.py
+++ b/lmcache/storage_backend/serde/cachegen_decoder.py
@@ -1,7 +1,7 @@
 from typing import List, Optional
 
 import torch
-import torchac_cuda  # type: ignore
+# import torchac_cuda  # type: ignore
 
 import lmcache.storage_backend.serde.cachegen_basics as CGBasics
 from lmcache.config import LMCacheEngineConfig, LMCacheEngineMetadata
