diff --git a/lmcache/storage_backend/serde/cachegen_decoder.py b/lmcache/storage_backend/serde/cachegen_decoder.py
index 470054a..2ee51ca 100644
--- a/lmcache/storage_backend/serde/cachegen_decoder.py
+++ b/lmcache/storage_backend/serde/cachegen_decoder.py
@@ -1,7 +1,6 @@
 from typing import List, Optional
 
 import torch
-import torchac_cuda  # type: ignore
 
 import lmcache.storage_backend.serde.cachegen_basics as CGBasics
 from lmcache.config import LMCacheEngineConfig, LMCacheEngineMetadata
diff --git a/setup.py b/setup.py
index 5b1aa4b..f1305b9 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,6 @@ setup(
         "nvtx",
         "safetensors",
         "transformers",
-        "torchac_cuda >= 0.2.5",
     ],
     classifiers=[
         # Trove classifiers
