diff --git a/src/parllama/utils.py b/src/parllama/utils.py
index da7e97f..b658f06 100644
--- a/src/parllama/utils.py
+++ b/src/parllama/utils.py
@@ -546,7 +546,7 @@ def clamp_input_value(
 eff_word_list_cache: list[str] = []
 
 
-def get_args() -> Namespace:
+def get_args(args: Optional[list[str]] = None) -> Namespace:
     """Parse and return the command line arguments.
 
     Returns:
@@ -664,7 +664,9 @@ def get_args() -> Namespace:
     )
 
     # Finally, parse the command line.
-    return parser.parse_args()
+    if args is None:
+        args = []
+    return parser.parse_args(args)
 
 
 def output_to_dicts(output: str) -> list[dict[str, Any]]: