diff --git a/testing/invariant/custom_types/matchers.py b/testing/invariant/custom_types/matchers.py
index ba1f316..6889302 100644
--- a/testing/invariant/custom_types/matchers.py
+++ b/testing/invariant/custom_types/matchers.py
@@ -1,6 +1,6 @@
 """Defines the expect functions."""
 
-from enum import StrEnum
+from enum import Enum
 from typing import Any
 
 from invariant.scorers.llm.classifier import Classifier
