diff --git a/testing/invariant/custom_types/matchers.py b/testing/invariant/custom_types/matchers.py
index ba1f316..d4a22cd 100644
--- a/testing/invariant/custom_types/matchers.py
+++ b/testing/invariant/custom_types/matchers.py
@@ -1,6 +1,6 @@
 """Defines the expect functions."""
 
-from enum import StrEnum
+from enum import Enum
 from typing import Any
 
 from invariant.scorers.llm.classifier import Classifier
@@ -86,7 +86,7 @@ class IsSimilar(Matcher):
 class IsFactuallyEqual(Matcher):
     """Matcher for checking if the output is close to expected using llm"""
 
-    class Agreement(StrEnum):
+    class Agreement(Enum):
         """Enum for different levels of agreement."""
 
         SUPER_STRICT_AGREEMENT = "super strict agreement"
diff --git a/testing/invariant/scorers/llm/clients/client.py b/testing/invariant/scorers/llm/clients/client.py
index 5c276f0..1dfe6d2 100644
--- a/testing/invariant/scorers/llm/clients/client.py
+++ b/testing/invariant/scorers/llm/clients/client.py
@@ -1,11 +1,11 @@
 """Abstract base class for LLM clients."""
 
 from abc import ABC, abstractmethod
-from enum import StrEnum
+from enum import Enum
 from typing import Any
 
 
-class SupportedClients(StrEnum):
+class SupportedClients(Enum):
     """Enumeration of supported LLM clients."""
 
     OPENAI = "OpenAI"
