diff --git a/rapid_doc/pdf_extract/main.py b/rapid_doc/pdf_extract/main.py
index b56818a..f66fd0f 100644
--- a/rapid_doc/pdf_extract/main.py
+++ b/rapid_doc/pdf_extract/main.py
@@ -19,8 +19,9 @@ from ..utils import is_contain_continous_str, only_contain_str
 
 
 class PDFExtract:
-    def __init__(self):
-        self.pdf_path = None
+    def __init__(self, pdf_path: str):
+        self.pdf_path = pdf_path
+        self.extract_all_pages(pdf_path)
         self.pages = None
         self.ratio = None
 
diff --git a/tests/test_files/direct_extract/single_column.pdf b/tests/test_files/direct_extract/single_column.pdf
index dad8138..62a69fb 100644
Binary files a/tests/test_files/direct_extract/single_column.pdf and b/tests/test_files/direct_extract/single_column.pdf differ
