diff --git a/pebblo/topic_classifier/topic_classifier.py b/pebblo/topic_classifier/topic_classifier.py
index 7b9453c..bc7e7eb 100644
--- a/pebblo/topic_classifier/topic_classifier.py
+++ b/pebblo/topic_classifier/topic_classifier.py
@@ -5,7 +5,7 @@ Defines the TopicClassifier class with methods for predicting topics and extract
 
 import os
 
-from huggingface_hub import login
+from huggingface_hub import HfApi
 from transformers import AutoModelForSequenceClassification, AutoTokenizer, pipeline
 
 from pebblo.log import get_logger
@@ -35,7 +35,7 @@ class TopicClassifier:
 
         # Check if the environment variable exists
         if huggingface_token is not None:
-            login(token=huggingface_token)
+            HfApi().set_access_token(huggingface_token)
 
         # Load the model and tokenizer from the specified paths and revision
         _tokenizer = AutoTokenizer.from_pretrained(