diff --git a/pyproject.toml b/pyproject.toml
index 2b23bd0..aab5f53 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -14,7 +14,7 @@ tqdm = "^4.66.2"
 rich = "^13.6.0"
 python-frontmatter = "^1.1.0"
 duckdb="0.10.0"
-unstructured="^0.12.6"
+unstructured={version=">=0.12.6,<0.13.0", python=">=3.10,<3.12"}
 pandas="2.2.1"
 instructor="^0.6.4"
 tenacity="^8.2.3"
@@ -23,6 +23,7 @@ scikit-learn = "^1.3.2"
 
 [tool.poetry.dev-dependencies]
 pytest = "^7.1.2"
+pytest-xdist = "^2.5.0"
 lancedb = "^0.6.1"
 
 [build-system]
