diff --git a/pyproject.toml b/pyproject.toml
index dfa9922..52a2c72 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -4,7 +4,6 @@ version = "0.0.0"
 dependencies = [
     "shodan",
     "dhooks",
-    "nmap",
     "colorama",
     "cryptography",
     "hashcat",
@@ -13,4 +12,4 @@ dependencies = [
 
 [build-system]
 requires = ["setuptools", "wheel"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
