diff --git a/Dockerfile b/Dockerfile
deleted file mode 100644
index a5f7173..0000000
--- a/Dockerfile
+++ /dev/null
@@ -1,9 +0,0 @@
-FROM python:3.11-slim
-
-RUN apt-get update && apt-get upgrade -y && rm -rf /var/lib/apt/lists/*
-
-RUN pip install --no-cache-dir scrapegraphai
-RUN pip install --no-cache-dir scrapegraphai[burr]
-
-RUN python3 -m playwright install-deps
-RUN python3 -m playwright install
\ No newline at end of file
diff --git a/scrapegraphai/graphs/smart_scraper_graph.py b/scrapegraphai/graphs/smart_scraper_graph.py
index cd9e75b..28f38d8 100644
--- a/scrapegraphai/graphs/smart_scraper_graph.py
+++ b/scrapegraphai/graphs/smart_scraper_graph.py
@@ -1,3 +1,5 @@
+class ScrapeGraph:
+    pass
 """
 SmartScraperGraph Module
 """
diff --git a/scrapegraphai/utils/__init__.py b/scrapegraphai/utils/__init__.py
index 22f6a4b..f85af58 100644
--- a/scrapegraphai/utils/__init__.py
+++ b/scrapegraphai/utils/__init__.py
@@ -2,6 +2,7 @@
     __init__.py file for utils folder
 """
 from .prettify_exec_info import prettify_exec_info
+from .data_export import convert_to_csv
 from .proxy_rotation import Proxy, parse_or_search_proxy, search_proxy_servers
 from .save_audio_from_bytes import save_audio_from_bytes
 from .sys_dynamic_import import dynamic_import, srcfile_import
diff --git a/scrapegraphai/utils/data_export.py b/scrapegraphai/utils/data_export.py
index fbff45e..bf0438a 100644
--- a/scrapegraphai/utils/data_export.py
+++ b/scrapegraphai/utils/data_export.py
@@ -1,3 +1,5 @@
+def convert_to_csv():
+    pass
 """
 data_export module 
 This module provides functions to export data to various file formats.
