diff --git a/ml_mdm/trainer.py b/ml_mdm/trainer.py
index fe1e0d1..74764fa 100644
--- a/ml_mdm/trainer.py
+++ b/ml_mdm/trainer.py
@@ -3,7 +3,8 @@
 import numpy as np
 import torch
 import torch.nn as nn
-from typing import Optional, Tuple 
+from typing import Optional, Tuple
+from tensorboard import SummaryWriter
 from argparse import Namespace
 
 def train_batch(
@@ -11,7 +12,7 @@ def train_batch(
     sample: dict,
     optimizer: torch.optim.Optimizer,
     scheduler: torch.optim.lr_scheduler.LRScheduler,
-    logger: Optional[torch.utils.tensorboard.SummaryWriter],
+    logger: Optional[SummaryWriter],
     args: Namespace,
     grad_scaler: Optional[torch.cuda.amp.GradScaler] = None,
     accumulate_gradient: bool = False,
