diff --git a/pyproject.toml b/pyproject.toml
index b0883f1..adc07d1 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -160,6 +160,7 @@ norecursedirs = [
     "docs",
 ]
 addopts = [
+    "--ignore=examples/getting_started/",
     "--strict-markers",
     "--doctest-modules",
     "--color=yes",
@@ -168,6 +169,8 @@ addopts = [
 ]
 markers = [
     "cloud: Run the cloud tests for example",
+    "timeout: Mark test to timeout",
+    "flaky: Mark test as flaky",
 ]
 filterwarnings = [
     "error::FutureWarning",
