diff --git a/src/auralis/models/xttsv2/XTTSv2.py b/src/auralis/models/xttsv2/XTTSv2.py
index 4add4cb..4049670 100644
--- a/src/auralis/models/xttsv2/XTTSv2.py
+++ b/src/auralis/models/xttsv2/XTTSv2.py
@@ -204,6 +204,7 @@ class XTTSv2Engine(BaseAsyncTTSEngine):
             device_map: Optional[str] = "auto",
             tensor_parallel_size: int = 1,
             pipeline_parallel_size: int = 1,
+            gpt_model: Optional[str] = "AstraMindAI/xtts2-gpt",
             **kwargs,
     ) -> nn.Module:
         """Load pretrained XTTS model from HuggingFace Hub."""
@@ -230,7 +231,7 @@ class XTTSv2Engine(BaseAsyncTTSEngine):
         hifi_config = XTTSConfig(**config)
 
         # Initialize model
-        model = cls(
+        model = cls(gpt_model=gpt_model,
             hifi_config=hifi_config,
             gpt_config=gpt_config,
             tensor_parallel_size=tensor_parallel_size,
