diff --git a/bigcodebench/sanitize.py b/bigcodebench/sanitize.py
index 2f8ba3b..370b7f3 100644
--- a/bigcodebench/sanitize.py
+++ b/bigcodebench/sanitize.py
@@ -1,3 +1,17 @@
+import os
+import pathlib
+from typing import Dict, Generator, List, Optional, Set, Tuple
+from pqdm.processes import pqdm
+from tqdm import tqdm
+import tree_sitter_python
+from tree_sitter import Language, Node, Parser
+from bigcodebench.data import (
+    get_bigcodebench,
+    load_solutions,
+    write_directory,
+    write_jsonl,
+)
+from bigcodebench.syncheck import syntax_check
 """Post-processing LLM-generated Python code implemented using tree-sitter."""
 
 import os
