diff --git a/setup.py b/setup.py
index c049b49..3f546e9 100644
--- a/setup.py
+++ b/setup.py
@@ -39,7 +39,6 @@ if __name__ == "__main__":
             "imageio",
             "imageio-ffmpeg",
             "optimum-quanto",
-            "flash_attn>=2.6.3"
         ],
         extras_require={
             "diffusers": [
diff --git a/xfuser/envs.py b/xfuser/envs.py
index 2a7e179..fb3dd4d 100644
--- a/xfuser/envs.py
+++ b/xfuser/envs.py
@@ -60,7 +60,7 @@ class PackagesEnvChecker:
 
     def initialize(self):
         self.packages_info = {
-            "has_flash_attn": self.check_flash_attn(),
+            "has_flash_attn": False,
             "has_long_ctx_attn": self.check_long_ctx_attn(),
             "diffusers_version": self.check_diffusers_version(),
         }
