diff --git a/setup.py b/setup.py
index c049b49..3f546e9 100644
--- a/setup.py
+++ b/setup.py
@@ -39,7 +39,6 @@ if __name__ == "__main__":
             "imageio",
             "imageio-ffmpeg",
             "optimum-quanto",
-            "flash_attn>=2.6.3"
         ],
         extras_require={
             "diffusers": [
diff --git a/xfuser/core/fast_attention/attn_layer.py b/xfuser/core/fast_attention/attn_layer.py
index e82c4bf..bda0247 100644
--- a/xfuser/core/fast_attention/attn_layer.py
+++ b/xfuser/core/fast_attention/attn_layer.py
@@ -7,7 +7,6 @@ import torch
 from diffusers.models.attention_processor import Attention
 from typing import Optional
 import torch.nn.functional as F
-import flash_attn
 from enum import Flag, auto
 from .fast_attn_state import get_fast_attn_window_size
 
diff --git a/xfuser/core/long_ctx_attention/ring/ring_flash_attn.py b/xfuser/core/long_ctx_attention/ring/ring_flash_attn.py
index 9e8b116..afbcb7b 100644
--- a/xfuser/core/long_ctx_attention/ring/ring_flash_attn.py
+++ b/xfuser/core/long_ctx_attention/ring/ring_flash_attn.py
@@ -1,6 +1,4 @@
 import torch
-import flash_attn
-from flash_attn.flash_attn_interface import _flash_attn_forward
 from xfuser.core.long_ctx_attention import xFuserLongContextAttention
 from xfuser.core.cache_manager.cache_manager import get_cache_manager
 from yunchang.ring.utils import RingComm, update_out_and_lse
diff --git a/xfuser/core/long_ctx_attention/ulysses/attn_layer.py b/xfuser/core/long_ctx_attention/ulysses/attn_layer.py
index ff8788f..3a01e7b 100644
--- a/xfuser/core/long_ctx_attention/ulysses/attn_layer.py
+++ b/xfuser/core/long_ctx_attention/ulysses/attn_layer.py
@@ -10,7 +10,7 @@ from yunchang.globals import PROCESS_GROUP
 from yunchang.comm.all_to_all import SeqAllToAll4D
 try:
     # yunchang > 0.4.0
-    from yunchang.kernels.attention import torch_attn
+    from yunchang.kernels.attention import pytorch_attn_forward as torch_attn
 except:
     print(f"detect you are not use the latest yunchang. Please install yunchang>=0.4.0")
     try:
diff --git a/xfuser/envs.py b/xfuser/envs.py
index 2a7e179..fb3dd4d 100644
--- a/xfuser/envs.py
+++ b/xfuser/envs.py
@@ -60,7 +60,7 @@ class PackagesEnvChecker:
 
     def initialize(self):
         self.packages_info = {
-            "has_flash_attn": self.check_flash_attn(),
+            "has_flash_attn": False,
             "has_long_ctx_attn": self.check_long_ctx_attn(),
             "diffusers_version": self.check_diffusers_version(),
         }
