diff --git a/constants/__init__.py b/constants/__init__.py
index 9cd0a26..cb863f9 100644
--- a/constants/__init__.py
+++ b/constants/__init__.py
@@ -4,6 +4,10 @@ from transformers import PreTrainedModel, LlamaForCausalLM, GemmaForCausalLM, St
 from typing import Type, Optional, Any, List, Tuple
 import math
 
+BLOCK_7B = "7B"
+ALLOWED_MODEL_TYPES_1 = ["type1", "type2"]
+ALLOWED_MODEL_TYPES_2 = ["type3", "type4"]
+
 @dataclass
 class CompetitionParameters:
     """Class defining model parameters"""
diff --git a/model/utils.py b/model/utils.py
index 93efeaa..7aae84a 100644
--- a/model/utils.py
+++ b/model/utils.py
@@ -1,9 +1,12 @@
 import base64
 import hashlib
 
+def get_model_criteria():
+    pass
+
 def get_hash_of_two_strings(string1: str, string2: str) -> str:
     """Hashes two strings together and returns the result."""
 
     string_hash = hashlib.sha256((string1 + string2).encode())
 
-    return base64.b64encode(string_hash.digest()).decode("utf-8")
\ No newline at end of file
+    return base64.b64encode(string_hash.digest()).decode("utf-8")
