diff --git a/setup.py b/setup.py
index c974ac5..9e2a12b 100644
--- a/setup.py
+++ b/setup.py
@@ -6,7 +6,7 @@ import os
 
 from setuptools import setup, find_packages
 
-install_requires = ['numpy>=1.17.2', 'torch>=1.7.0', 'scipy==1.6.0', 'pandas>=1.0.5', 'tqdm>=4.48.2',
+install_requires = ['numpy>=1.17.2', 'torch>=1.7.0', 'scipy==1.7.3', 'pandas>=1.0.5', 'tqdm>=4.48.2',
                     'colorlog==4.7.2','colorama==0.4.4',
                     'scikit_learn>=0.23.2', 'pyyaml>=5.1.0', 'tensorboard>=2.5.0']
 