diff --git a/hydra/modules/hydra.py b/hydra/modules/hydra.py
index 5dc5f62..8041b90 100755
--- a/hydra/modules/hydra.py
+++ b/hydra/modules/hydra.py
@@ -13,7 +13,7 @@ try:
 except ImportError:
     RMSNormGated = None
 
-from mamba_ssm.ops.triton.ssd_combined import mamba_chunk_scan_combined
+# from mamba_ssm.ops.triton.ssd_combined import mamba_chunk_scan_combined
 
 from hydra.modules.ops import hydra_split_conv1d_scan_combined
 
diff --git a/hydra/modules/matrix_mixers/quasiseparable.py b/hydra/modules/matrix_mixers/quasiseparable.py
index a56203b..55dcb3f 100755
--- a/hydra/modules/matrix_mixers/quasiseparable.py
+++ b/hydra/modules/matrix_mixers/quasiseparable.py
@@ -7,7 +7,7 @@ import torch.nn.functional as F
 
 from einops import rearrange, repeat
 
-from mamba_ssm.ops.triton.ssd_combined import mamba_chunk_scan_combined
+# from mamba_ssm.ops.triton.ssd_combined import mamba_chunk_scan_combined
 
 
 class Quasiseparable(nn.Module):
diff --git a/hydra/modules/ops.py b/hydra/modules/ops.py
index 13e5279..58fd355 100644
--- a/hydra/modules/ops.py
+++ b/hydra/modules/ops.py
@@ -1,14 +1,14 @@
 import torch
 import torch.nn.functional as F
 from einops import einsum, rearrange, repeat
-from mamba_ssm.ops.triton.layernorm_gated import (
-    _layer_norm_fwd,
-    _layer_norm_bwd,
-)
-from mamba_ssm.ops.triton.ssd_combined import (
-    _mamba_chunk_scan_combined_bwd,
-    _mamba_chunk_scan_combined_fwd,
-)
+# from mamba_ssm.ops.triton.layernorm_gated import (
+#     _layer_norm_fwd,
+#     _layer_norm_bwd,
+# )
+# from mamba_ssm.ops.triton.ssd_combined import (
+#     _mamba_chunk_scan_combined_bwd,
+#     _mamba_chunk_scan_combined_fwd,
+# )
 
 
 def flip(input):  # just for brevity