diff --git a/hydra/modules/hydra.py b/hydra/modules/hydra.py
index 5dc5f62..8041b90 100755
--- a/hydra/modules/hydra.py
+++ b/hydra/modules/hydra.py
@@ -13,7 +13,7 @@ try:
 except ImportError:
     RMSNormGated = None
 
-from mamba_ssm.ops.triton.ssd_combined import mamba_chunk_scan_combined
+# from mamba_ssm.ops.triton.ssd_combined import mamba_chunk_scan_combined
 
 from hydra.modules.ops import hydra_split_conv1d_scan_combined
 
