diff --git a/examples/YOLOv8-OpenCV-ONNX-Python/main.py b/examples/YOLOv8-OpenCV-ONNX-Python/main.py
index 8d9c7a50..f7a53698 100644
--- a/examples/YOLOv8-OpenCV-ONNX-Python/main.py
+++ b/examples/YOLOv8-OpenCV-ONNX-Python/main.py
@@ -5,7 +5,7 @@ import argparse
 import cv2.dnn
 import numpy as np
 
-from ultralytics.utils import ASSETS, yaml_load
+from examples.ultralytics.utils import ASSETS, yaml_load
 from ultralytics.utils.checks import check_yaml
 
 CLASSES = yaml_load(check_yaml("coco128.yaml"))["names"]
diff --git a/examples/YOLOv8-OpenCV-int8-tflite-Python/main.py b/examples/YOLOv8-OpenCV-int8-tflite-Python/main.py
index 53fba1f5..510d533c 100644
--- a/examples/YOLOv8-OpenCV-int8-tflite-Python/main.py
+++ b/examples/YOLOv8-OpenCV-int8-tflite-Python/main.py
@@ -6,7 +6,7 @@ import cv2
 import numpy as np
 from tflite_runtime import interpreter as tflite
 
-from ultralytics.utils import ASSETS, yaml_load
+from examples.ultralytics.utils import ASSETS, yaml_load
 from ultralytics.utils.checks import check_yaml
 
 # Declare as global variables, can be updated based trained model image size
diff --git a/examples/YOLOv8-SAHI-Inference-Video/yolov8_sahi.py b/examples/YOLOv8-SAHI-Inference-Video/yolov8_sahi.py
index f2b8274c..8e4903b1 100644
--- a/examples/YOLOv8-SAHI-Inference-Video/yolov8_sahi.py
+++ b/examples/YOLOv8-SAHI-Inference-Video/yolov8_sahi.py
@@ -6,7 +6,7 @@ from pathlib import Path
 import cv2
 from sahi import AutoDetectionModel
 from sahi.predict import get_sliced_prediction
-from sahi.utils.yolov8 import download_yolov8s_model
+from sahi.utils.yolov8onnx import download_yolov8n_onnx_model as download_yolov8s_model
 
 from ultralytics.utils.files import increment_path
 
diff --git a/examples/YOLOv8-Segmentation-ONNXRuntime-Python/main.py b/examples/YOLOv8-Segmentation-ONNXRuntime-Python/main.py
index 141d21b9..0fd77975 100644
--- a/examples/YOLOv8-Segmentation-ONNXRuntime-Python/main.py
+++ b/examples/YOLOv8-Segmentation-ONNXRuntime-Python/main.py
@@ -6,7 +6,7 @@ import cv2
 import numpy as np
 import onnxruntime as ort
 
-from ultralytics.utils import ASSETS, yaml_load
+from examples.ultralytics.utils import ASSETS, yaml_load
 from ultralytics.utils.checks import check_yaml
 from ultralytics.utils.plotting import Colors
 
